.github/workflows/scripts/firestore.rules

```diff
@@ -0,0 +1,8 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /{document=**} {
+      allow read, write: if true;
```

Comment:
Are these going to be visible to anyone outside of the repo? If so, we should strongly prefer having stronger rules than these.

Reply:
@puf This has been the motivation I needed to actually learn about the security rules, but I have a question. It seems that no other snippets projects are defining rules in their project code, so I'm assuming they're solely using the console. Do you have a recommendation or preference on which route to take? I'm only using the files in the code because initializing the project generated the files.